.circleci/getChangelogs.sh

```diff
@@ -1,28 +1,41 @@
 #!/bin/bash
 set -e
-REQUIREMENTS_FILE="codacy/requirements.yaml"
-OLD_LOCK_FILE="codacy/requirements_old.lock"
-NEW_LOCK_FILE="codacy/requirements.lock"
+REQUIREMENTS_FILE="$(pwd)/codacy/requirements.yaml"
+OLD_LOCK_FILE="$(pwd)/codacy/requirements_old.lock"
+NEW_LOCK_FILE="$(pwd)/codacy/requirements.lock"
+CHANGELOG_FILE="$(pwd)/changelogs/changelog.md"
 
-function prepareChangelogMarkdown(){
+function appendToChangelog() {
+    echo "$1" >> "$CHANGELOG_FILE"
+}
+
+function prepareEnvironment() {
+    rm -rf ./changelogs
+    mkdir changelogs
+    prepareChangelogMarkdown
+    git cat-file blob "latest":"codacy/requirements.lock" > "$OLD_LOCK_FILE"
+}
+
+function prepareChangelogMarkdown() {
     changelog_conf_path=$(realpath .)/.changelogs
     export GITCHANGELOG_CONFIG_FILENAME=$changelog_conf_path/gitchangelog.rc
     markdown_template_path=$(echo "$changelog_conf_path/markdown.tpl" | sed 's/\//\\\//g')
     sed "s/##PATHMACRO##/$markdown_template_path/g" "$changelog_conf_path/gitchangelogtemplate.rc" >> "$GITCHANGELOG_CONFIG_FILENAME"
-    echo "# Codacy Chart Changelog" >> "changelogs/changelog.md"
+    appendToChangelog "# Codacy Chart Changelog"
 }
 
 function getChangelog() {
     project_name=$1
     old_version=$2
     new_version=$3
     repository_url=$4
-    echo " * getting changelog ..."
     cd "$project_name"
+    echo "Getting changelog: $old_version -> $new_version"
     git fetch --all --quiet
     git checkout tags/"$old_version" --quiet
-    echo "## $project_name($repository_url)" >> "../changelogs/changelog.md"
-    gitchangelog "$old_version" "$new_version" >> "../changelogs/changelog.md"
+    appendToChangelog "## $project_name([$repository_url]())"
```

Comment:
What do you think of adding this space between the project name and the repository URL for easier readability?

```suggestion
    appendToChangelog "## $project_name ([$repository_url]())"
```